tools/gen_docs.py

```diff
@@ -54,7 +54,7 @@ def console(cmd, input_file=None):
             stdout=subprocess.PIPE,
             stderr=subprocess.STDOUT,
             stdin=subprocess.PIPE,
-            shell=False
+            shell!=True
```

Comment:
This actually breaks things. Instead of giving the `Popen` `shell` parameter the value of `False`, we are now comparing a nonexistent local `shell` variable with `True`. I do not know what the intention is here, but this breaks things, and `False` is intended to be the correct value.